### PR TITLE
Scenario update

### PR DESCRIPTION
**Pull request by solonarieugeniu**
Natalia, please review and approve

### CODE REVIEW

File: src/test/resources/Example.feature [hunk 1]
```diff
@@ -4,4 +4,6 @@
 Feature: A description
 
   Scenario: A scenario
-    Given something..
\ No newline at end of file
+    Given something..
+    When click
+    Then 200 OK
```
Comment (NataliaBodrug) [inline src/test/resources/Example.feature]: please new line;

File: src/test/resources/Example.feature [hunk 2]
```diff
@@ -4,4 +4,6 @@
 Feature: A description
 
   Scenario: A scenario
```
Comment (NataliaBodrug) [inline src/test/resources/Example.feature:6]: please add un tag

**Review by NataliaBodrug — COMMENTED** [review]
Please see comments

File: src/test/resources/Example.feature (hunk 1 above)
Comment (solonarieugeniu) [reply src/test/resources/Example.feature]: updated, thanks

File: src/test/resources/Example.feature (hunk 2 above)
Comment (solonarieugeniu) [reply src/test/resources/Example.feature:6]: nu sunt de-acord Natalia )

**Review by NataliaBodrug — APPROVED** [review]

**Pull request merged**